project/core/settings.py

```diff
@@ -7,7 +7,7 @@
 import os
 
 # False if not in os.environ
-DEBUG = os.getenv("DEBUG", False)
+DEBUG = os.getenv("DEBUG", True)
```

Comment:
We probably don't want to default DEBUG to true, since it is only used for development.

```suggestion
DEBUG = os.getenv("DEBUG", False)
```